manifests/plugin/ansible.pp

```diff
@@ -38,6 +38,12 @@
     template_path => 'foreman_proxy/plugin/ansible.yml.erb',
   }
 
+  file { '/usr/share/foreman-proxy/.ansible':
```

Comment:
Like @mmoll said, this should be ```"${::foreman_proxy::dir}```

Reply:
as @lpramuk mentioned we also need .ansible_galaxy for jobs that install roles from galaxy

Reply:
```
~foreman-proxy/.ansible
~foreman-proxy/.ansible_galaxy
```
we need both, the latter is for galaxy jobs